docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>
+### Use Input properties in TextField
+
+Input properties, such as `InputColor` and `InputPointSize`, are used when you want to set new input texts differently from the existing text settings when the new text inputs.
+
+Especially, these properties should be used with some specific conditions.

```

Comment:
Remove this line.

Reply:
This sentence is a key point. Input properties should be used with signals or any conditions.

Reply:
@Seoyeon2Kim, Thanks for clarifying. I understand that this section is for entire set of Input properties and not just for InputColor and InputPointSize, and also about the specific conditions. Here is my suggestion:

To change the text settings for new input text, you can use the Input properties of the [Tizen.NUI.BaseComponents.TextField](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextField.html) class, such as `InputColor`, `InputPointSize` and so on.

To use these properties there are some specific conditions. For instance, you can use these properties either when the `TextChanged` event occurs or when the `Clicked` event of Button occurs.

Reply:
Okay, Changed as your suggestion.